app/models/room.rb

```diff
@@ -1,4 +1,11 @@
 class Room < ApplicationRecord
   has_many :reservations
   has_many :users, through: :reservations
+
+  validates :name, presence: true
+  validates :stars, presence: true
+  validates :persons_allowed, presence: true
+  validates :photo, presence: true
+  validates :description, presence: true
+  validates :price, presence: true
```

Comment:
It would be good to have some format and type validations to ensure security in the database.

Reply:
👍